documentation/book/assembly-getting-started-strimzi-admin.adoc

```diff
@@ -0,0 +1,13 @@
+// This assembly is included in the following assemblies:
+//
+// getting-started.adoc
+
+[id='assembly-getting-started-strimzi-admin-{context}']
+= Strimzi Administrator
+
```

Comment:
_Please reword this concept module as follows:_

{ProductName} includes several custom resources. By default, permission to create, edit, and delete these resources is limited to {ProductPlatformName} cluster administrators. If you want to allow non-cluster administators to manage {ProductName} resources, you must assign them the Strimzi Administrator role.